### PR TITLE
profile styles

### PR DESCRIPTION
**Pull request by stormcloud266**
Screenshot 2024-07-08 at 4 45 09 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#550** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/550?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/profile/screen.test.tsx [hunk 1]
```diff
@@ -41,6 +42,32 @@ jest.mock('app/utils/useUser', () => ({
...
+    error: null,
+    isFetching: false,
+    isFetchingNextPage: false,
+    fetchNextPage: () => {},
+    hasNextPage: false,
+    data: {
+      pages: [
+        [
+          {
+            created_at: new Date('2024-5-19 GMT'),
```
Comment (0xBigBoss) [inline packages/app/features/profile/screen.test.tsx:57]: nice. TIL

File: packages/app/features/profile/screen.tsx [hunk 2]
```diff
@@ -1,76 +1,297 @@
...
-          ) : null}
-        </YStack>
-      ) : null}
-    </YStack>
+    <>
+      <YStack f={1} gap="$6">
+        {error && <Text theme="red">{error.message}</Text>}
+        {isLoading && <Spinner size="large" color="$color10" />}
+        {otherUserProfile ? (
+          <YStack h={media.shorter ? '83vh' : '88vh'} $gtMd={{ height: 'auto' }}>
```
Comment (0xBigBoss) [inline packages/app/features/profile/screen.tsx:61]: we will need to start making things compatible with native very soon.

```suggestion
          <YStack h={isWeb ? (media.shorter ? '83vh' : '88vh') : '83%'} $gtMd={{ height: 'auto' }}>
```

File: packages/app/features/profile/utils/useInterUserActivityFeed.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+} from '@tanstack/react-query'
+import { useSupabase } from 'app/utils/supabase/useSupabase'
+import { throwIf } from 'app/utils/throwIf'
+import { EventArraySchema, type Activity } from 'app/utils/zod/activity'
+import type { ZodError } from 'zod'
+
+/**
+ * Infinite query to fetch ERC-20 token activity feed between the current user and another profile.
+ *
+ * @note does not support ETH transfers. Need to add another shovel integration to handle ETH receives, and another one for ETH sends
```
Comment (0xBigBoss) [inline packages/app/features/profile/utils/useInterUserActivityFeed.ts:15]: can remove 😅 i added this awhile ago.

File: packages/app/features/profile/screen.tsx [hunk 4]
```diff
@@ -1,76 +1,297 @@
...
 
   return (
-    <YStack f={1} gap="$6" pt="$6" $gtMd={{ pt: '$0' }}>
-      {error && <Text theme="red">{error.message}</Text>}
-      {isLoading && <Spinner size="large" color="$color10" />}
-      {profile ? (
-        <YStack width="100%" gap="$2">
-          <AvatarProfile profile={profile} /> <H1 nativeID="profileName">{profile.name}</H1>
-          <H2 theme="alt1">{formatTags(profile.all_tags ?? [])}</H2>
-          <Paragraph mb="$4">{profile.about}</Paragraph>
```
Comment (0xBigBoss) [inline packages/app/features/profile/screen.tsx:40]: hmm can we bring this back? looks like we missed this in the designs. Just make your best judgement on where to put it.

File: packages/playwright/tests/fixtures/profiles/test.ts [hunk 5]
```diff
@@ -20,7 +18,6 @@ export class ProfilePage {
     expect?.(title).toBe('Send | Profile')
     await expect?.(this.page.getByRole('heading', { name: tag })).toBeVisible()
     await expect?.(this.page.locator('#profileName')).toHaveText(this.profile.name)
-    await expect?.(this.page.getByText(this.profile.about, { exact: true })).toBeVisible()
```
Comment (0xBigBoss) [inline packages/playwright/tests/fixtures/profiles/test.ts:23]: should leave this in too.

File: packages/playwright/tests/profile.onboarded.spec.ts
Comment (0xBigBoss) [inline packages/playwright/tests/profile.onboarded.spec.ts:1]: If you are up to it, would be nice to add a few assertions that the activity renders correctly on this page. You can see how it's done in another test here, https://github.com/0xsend/sendapp/blob/bc056adeee8f5f75d33b3c7c71b6b611dc550e97/packages/playwright/tests/activity.onboarded.spec.ts#L27.

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/profile/screen.tsx (hunk 2 above)
Comment (stormcloud266) [reply packages/app/features/profile/screen.tsx:61]: yes i forgot this wasn't supported on native

File: packages/playwright/tests/profile.onboarded.spec.ts
Comment (stormcloud266) [reply packages/playwright/tests/profile.onboarded.spec.ts:1]: Awesome i'll mess around with this

File: packages/app/features/profile/utils/useInterUserActivityFeed.ts (hunk 3 above)
Comment (stormcloud266) [reply packages/app/features/profile/utils/useInterUserActivityFeed.ts:15]: Oh okay, well the eth transactions aren't showing. is there something i have to do to get them working?

File: packages/playwright/tests/profile.onboarded.spec.ts
Comment (0xBigBoss) [reply packages/playwright/tests/profile.onboarded.spec.ts:1]: Hopefully this helps! Sry for rambling in the video https://send-app.slack.com/archives/C07BGR3DJ8G/p1720559210421469

**Pull request merged**